src/_guides/language/effective-dart/usage.md

```diff
@@ -938,16 +938,16 @@ set x(num value) => center = Point(value, center.y);
 {% endprettify %}
 
 
-### DON'T use `this.` when not needed to avoid shadowing.
+### DON'T use `this.` except to avoid shadowing or redirect to a named constructor.
```

Comment:
I'm tempted to say or redirect -> or to redirect (since I first read it as "to avoid redirect", but then we have so many "to"s in a row...

Reply:
We need to update the big list o' links in https://www.dartlang.org/guides/language/effective-dart#usage. To keep old links working, we should also specify an anchor that's the same as the old link.

An example of doing this is in https://github.com/dart-lang/site-www/pull/1323.